### ISSUE TITLE
No fonts detected in SVG file due to missing trailing semicolon after font-family:

### ISSUE THREAD

**Issue by GrahamHannington** [phauer/svg-buddy#2]
A snippet from the input SVG file (created by CorelDRAW 2019):

````svg
 <defs>
  <style type="text/css">
   <![CDATA[

    /* ... */

    .fnt2 {font-weight:normal;font-size:141.06px;font-family:'IBM Plex Mono'}
    .fnt0 {font-weight:normal;font-size:141.06px;font-family:'IBM Plex Sans'}
    .fnt5 {font-weight:bold;font-size:141.06px;font-family:'IBM Plex Mono'}
    .fnt1 {font-weight:bold;font-size:141.06px;font-family:'IBM Plex Sans'}
    .fnt3 {font-style:italic;font-weight:normal;font-size:141.06px;font-family:'IBM Plex Mono'}
    .fnt4 {font-style:italic;font-weight:normal;font-size:141.06px;font-family:'IBM Plex Sans'}
   ]]>
  </style>
 </defs>
````

`svg-buddy` 1.2.2 reports "No fonts detected".

After looking at the relevant regex in [`SvgFontDetector.kt`](https://github.com/phauer/svg-buddy/blob/8c3bc5f8ea93d734e872aec8dd7cda46291a6541/src/main/kotlin/com/phauer/svgbuddy/processing/SvgFontDetector.kt#L16), I can see why:

````java
val fonts1 = detectusedFontsWithRegex(inputSvgString, Pattern.compile("font-family:(.*?);"))
````

My input SVG is missing the expected trailing semicolon after `font-family:`.

Sure enough, after I edited the input SVG file and appended the missing semicolons, `svg-buddy` detected the fonts.

For some thoughts on those semicolons, see the Stack Overflow question "[Leaving out the last semicolon of a CSS block](https://stackoverflow.com/questions/11939595/leaving-out-the-last-semicolon-of-a-css-block)".

**Comment by phauer** [comment]
Thank you for reporting this issue. I just released version 1.2.3 containing the fix. :-)

**Comment by GrahamHannington** [comment]
You're welcome. Thank you for `svg-buddy`!
